.github/workflows/build.yaml

```
@@ -0,0 +1,25 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
```

```suggestion
  push:
    branches:
      - '*'
```

We actually want this to run for all branches. Your current config will not work if there is a pull request to a branch other than the `main` branch, which can happen if multiple people are working on the same feature.